common/djangoapps/student/views.py

```diff
@@ -693,6 +703,47 @@ def _create_recent_enrollment_message(course_enrollment_pairs, course_modes):
         )
 
 
+def _create_credit_availability_message(username, enrolled_courses_dict, user):
+    """Builds a dict of credit availability for courses.
+
+    Construct a for courses user has completed and has not purchased credit
+    from the credit provider yet.
+
+    Args:
+        username (str): Username of the user.
+        course_enrollment_pairs (list): A list of tuples containing courses, and the associated enrollment information.
+
+    Returns:
+        A dict of courses user is eligible for credit.
+
+    """
+    user_eligibitlites = get_credit_eligibility(username=username)
+    user_purchased_credit = get_purchased_credit_courses(username=username)
+
+    eligibility_messages = {}
+    for course_id, eligibility in user_eligibitlites.iteritems():
+        if course_id not in user_purchased_credit:
+            duration = eligibility["seconds_good_for_display"]
+            curr_time = timezone.now()
+            validity_till = eligibility["created_at"] + timedelta(seconds=duration)
+            if validity_till > curr_time:
+                diff = validity_till - curr_time
+                urgent = diff.days <= 30
+                eligibility_messages[course_id] = {
+                    "course_id": course_id,
+                    "course_name": enrolled_courses_dict[course_id].display_name,
+                    "is_eligible": True,
```

Comment:
What is use of this hard coded variable `is_eligible`?
Wouldn't `eligibility_message` itself means that user is eligible?